CONTRIBUTING.md

```diff
@@ -23,7 +23,7 @@ To test examples, run `test_examples.sh [gpu device id]`. `-1` would run example
 
 We use PEP8. To check your code, use the `black`, `isort`, and `flake8` packages.
 ```
-$ pip install git+git://github.com/psf/black.git flake8 mypy isort
+$ pip install black@git+https://github.com/psf/black.git flake8 mypy isort
```

Comment:
Is there a specific reason for installing `black` from the original repository?
If not, I suggest installing it via PyPI:
```suggestion
$ pip install black flake8 mypy isort
```